test/server_integration/http/platform/status.ts

```diff
@@ -53,6 +56,7 @@ export default function ({ getService }: FtrProviderContext) {
         5_000,
         async () => (await getStatus('statusPluginA')).level === 'degraded'
       );
+      await statusPropagation();
       expect((await getStatus('statusPluginA')).level).to.eql('degraded');
       expect((await getStatus('statusPluginB')).level).to.eql('degraded');
```

Comment:
Same comment than https://github.com/elastic/kibana/pull/113214#discussion_r718409911, still not sure how the PR changes introduced the added required delay, but...